### PR TITLE
DRIVER signup form text updates #133

### PR DESCRIPTION
**Pull request by joesweeny**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -311,14 +311,16 @@ <h3 class="bannerbox__title">I can offer a ride</h3>
                         <a class="close-form button--cancel" href="#intro">&times;</a>
 
                         <fieldset class="date-time-pickers">
-                            <legend>When and where can you pick someone up?</legend>
+                            <legend>What can you offer?</legend>
 
                             <div class="form-group date-container">
-                                <label for="driver-date">Date available</label>
+                                <label for="driver-date">Dates available</label>
+                                <p>On what date(s) would you be a available to give rides?" We'll propose matches for all of the dates that you pick.</p>
```
Comment (richardwestenra) [inline index.html]: It looks like the extra double quotation mark after `give rides?` was mistakenly included in the copy supplied in the issue. It doesn't make sense here, so do you mind removing it please? Sorry, this is not your fault.

CC @sasjkia

File: index.html [hunk 2]
```diff
@@ -338,8 +340,8 @@ <h3 class="bannerbox__title">I can offer a ride</h3>
                             <legend>Location and vehicle</legend>
 
                             <div class="form-group">
-                                <label for="rideArea">Collection ZIP code</label>
-                                <input type="text" class="form-input" pattern="(^\d{5}$)|(^\d{5}-\d{4}$)" id="offerArea" placeholder="Where you can collect the rider" name="DriverCollectionZIP" required>
+                                <label for="rideArea">Pick up zip code</label>
```
Comment (richardwestenra) [inline index.html]: Can you please capitalise `ZIP`?

File: index.html [hunk 3]
```diff
@@ -391,7 +393,8 @@ <h3 class="bannerbox__title">I can offer a ride</h3>
 
                         <div class="form-group checkbox">
                             <label for="officialDriver">
-                                <input type="checkbox" id="officialDriver" name="DrivingOnBehalfOfOrganization" /> I am driving in an official capacity on behalf of a registered organization
+                              <input type="checkbox" class="hidden" name="V" value="B" style="display:none;">
+                                <input type="checkbox" id="officialDriver" name="DrivingOnBehalfOfOrganization" style="display:none;" class="hidden" /> <!--I am driving in an official capacity on behalf of a registered organization -->
```
Comment (richardwestenra) [inline index.html]: I'm not sure that this is what was intended, but I'll need to confirm with @sasjkia what she meant when she said to hide this checkbox. I assume she meant to delete this form row. Sorry, I'll get back to you :(

Comment (richardwestenra) [inline index.html]: okay so here's the plan: 

- Can you please delete the `officialDriver` AND `quesDriverOrgName` form group rows? i.e. the following code:
```
<div class="form-group checkbox">
    <label for="officialDriver">
        <input type="checkbox" id="officialDriver" name="DrivingOnBehalfOfOrganization" /> I am driving in an official capacity on behalf of a registered organization
    </label>
</div>
<div class="form-group" id="quesDriverOrgName" style="display:none">
    <label for="officialDriverOrgName">Organization name</label>
    <input type="text" class="form-input" id="officialDriverOrgName" name="DrivingOBOOrganizationName" data-error="The organisation name is required if the above box is checked.">
    <div class="help-block with-errors"></div>
</div>
```
- Please also delete the JS listener which toggles the org name row on/off:
```
// Toggle Official Driver input when checkbox is ticked:
$('#officialDriver').on('change', function() {
    var isOfficialDriver = $(this).is(':checked');
    $('#quesDriverOrgName').toggle(isOfficialDriver)
        .find('#officialDriverOrgName').prop('required', isOfficialDriver);
});
```

**Comment by joesweeny** [comment]
@richardwestenra all issues fixed my friend. Apologies if I caused any confusion but I hope this should all be sorted now

**Comment by richardwestenra** [comment]
Perfect, thanks! 🚢 

Nice work :)

**Pull request merged**